### PR TITLE
Correct some typos for MultipleSequenceAlignment.rst

### PR DESCRIPTION
**Pull request by sfchen**
Like:
coputing --> computing
usind --> using

### CODE REVIEW

**Comment by sfchen** [comment]
Seems travis CI is broken?

**Comment by h-2** [comment]
Thank you for the PR! But, can you please rebase and re-open it against the develop-branch?

The Windows-Jenkins builds are known to be broken, but travis should work. We will look into it. In any case, can you please also sign the CLA? → http://www.clahub.com/agreements/seqan/seqan
Thanks!

**Comment by sfchen** [comment]
I just changed the base to develop branch.

But I failed to sign the CLA. An "Application Error" was encountered after I filled the form and clicked "I Agree".

**Comment by h-2** [comment]
> But I failed to sign the CLA. An "Application Error" was encountered after I filled the form and clicked "I Agree".

Really? Could you try again? Sorry about this, we had some problems with clahub in the beginning, but then it seemed to work reliably.

File: manual/source/Tutorial/Algorithms/Alignment/MultipleSequenceAlignment.rst [hunk 1]
```diff
@@ -61,7 +61,7 @@ The output of the program look as follows.
 .. includefrags:: demos/tutorial/multiple_sequence_alignment/msa.cpp.stdout
 
 Note that we stored the MSA in an :dox:`Align` object which allows easy access to the individual rows of the MSA as :dox:`Gaps` objects.
-:dox:`globalMsaAlignment` also allows storing the alignment as an :dox:`AlignmentGraph`.
+:dox:`globalMsaAlignment` also allows storing the alignment as a :dox:`AlignmentGraph`.
```
Comment (h-2) [inline manual/source/Tutorial/Algorithms/Alignment/MultipleSequenceAlignment.rst]: really? 🤔

**Comment by smehringer** [comment]
@sfchen thank you for the PR and sorry that we did not work on this for so long. 
Could you ckean up the commit history to only have your commit and include h-2's comment (I think it is 'an' AlignmentGraph)?

**Comment by smehringer** [comment]
@sfchen ping

**Comment by sfchen** [comment]
Hi, I think it's better to close this PR first since this is a so tiny change:)

**Pull request closed without merging**